src/box/schema.cc

```diff
@@ -296,6 +306,9 @@ sc_space_new(uint32_t id, const char *name,
 	space_cache_replace(NULL, space);
 	if (replace_trigger)
 		trigger_add(&space->on_replace, replace_trigger);
+	struct trigger *t = (struct trigger *) malloc(sizeof(*t));
```

Comment:
may be `diag_raise` or `panic` in case malloc fails? Also i have never seen where we destoy this trigger.

Reply:
fixed